ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/snmp-settings.j2

```diff
@@ -1,6 +1,21 @@
 {# eos - SNMP Settings#}
 {% if snmp_server is arista.avd.defined %}
 !
+{%     if snmp_server.engine_ids is arista.avd.defined %}
+{%         if snmp_server.engine_ids.local is arista.avd.defined %}
+snmp-server engineID local {{ snmp_server.engine_ids.local }}
+{%         endif %}
+{%         for engine_id in snmp_server.engine_ids.remotes %}
+{%             if engine_id.id is arista.avd.defined and engine_id.address is arista.avd.defined %}
+{%                 set remote_engine_ids_cli =  "snmp-server engineID remote " ~ engine_id.address %}
+{%                 if engine_id.port is arista.avd.defined %}
```

Comment:
Should this knob be ```port``` or ```udp_port``` to match the cli? Any suggestions @ClausHolbechArista @carlbuchmann

Reply:
IMO, udp_port sounds better.

Reply:
I think we had udp_port under the users, but changed it to port to match the engine data model. If we change to udp_port, please change both engine_id and users models

Reply:
The users model is having ```udp_port```.